components/library/gd2/gd2.p5m

```diff
@@ -96,18 +96,17 @@ file path=usr/include/gd2/gdfontmb.h
 file path=usr/include/gd2/gdfonts.h
 file path=usr/include/gd2/gdfontt.h
 file path=usr/include/gd2/gdfx.h
-file path=usr/include/gd2/gdhelpers.h
-file path=usr/include/gd2/jisx0208.h
-file path=usr/include/gd2/wbmp.h
+file src/gdhelpers.h path=usr/include/gd2/gdhelpers.h
+file src/jisx0208.h path=usr/include/gd2/jisx0208.h
+file src/wbmp.h path=usr/include/gd2/wbmp.h
 
-file path=usr/lib/$(MACH64)/libgd.so.2.0.0
+file path=usr/lib/$(MACH64)/libgd.so.3.0.0
 file path=usr/lib/$(MACH64)/llib-lgd.ln
-file path=usr/lib/libgd.so.2.0.0
+file path=usr/lib/libgd.so.3.0.0
 file llib-lgd path=usr/lib/llib-lgd
 file path=usr/lib/llib-lgd.ln
 
-file entities.html path=usr/share/doc/gd2/html/entities.html
-file index.html path=usr/share/doc/gd2/html/index.html
+file src/entities.html path=usr/share/doc/gd2/html/entities.html
```

Comment:
No idea, this is part of original recipe. index is no longer present anywhere for libgd-2.1..